components/webac/src/main/java/org/trellisldp/webac/WebAcFilter.java

```diff
@@ -149,7 +154,22 @@ private WebAcFilter(final WebAcService accessService, final Config config) {
                 asList(config.getOptionalValue(CONFIG_WEBAC_CHALLENGES, String.class).orElse("").split(",")),
                 config.getOptionalValue(CONFIG_WEBAC_REALM, String.class).orElse("trellis"),
                 config.getOptionalValue(CONFIG_WEBAC_SCOPE, String.class).orElse(""),
-                config.getOptionalValue(CONFIG_HTTP_BASE_URL, String.class).orElse(null));
+                config.getOptionalValue(CONFIG_HTTP_BASE_URL, String.class).orElse(null),
+                config.getOptionalValue(CONFIG_WEBAC_ENABED, Boolean.class).orElse(Boolean.TRUE));
+    }
+
+    /**
+     * Create a WebAc-based auth filter.
+     *
+     * @param accessService the access service
+     * @param challengeTypes the WWW-Authenticate challenge types
+     * @param realm the authentication realm
+     * @param scope the authentication scope
+     * @param baseUrl the base URL, may be null
+     */
+    public WebAcFilter(final WebAcService accessService, final List<String> challengeTypes,
```

Comment:
The next release will be a breaking change anyway, so there's not a huge need to add this ctor (perhaps add a `@Deprecated` annotation so that we can remove it entirely later)

Reply:
Added the deprecated annotation and a comment.